### PR TITLE
fix deb package version

### PR DESCRIPTION
**Pull request by crazy-max**
follow-up https://github.com/docker/docker-ce-packaging/pull/701

### CODE REVIEW

**Comment by crazy-max** [comment]
```
./pkg/containerd/bin
└── [4.0K]  debian
    └── [4.0K]  bookworm
        └── [4.0K]  amd64
            ├── [2.8M]  containerd.io-dbgsym_0.0.0~20230330082121~812111e-1~debian.12~bookworm_amd64.deb
            ├── [5.2K]  containerd.io_0.0.0~20230330082121~812111e-1~debian.12~bookworm_amd64.buildinfo
            ├── [1.8K]  containerd.io_0.0.0~20230330082121~812111e-1~debian.12~bookworm_amd64.changes
            ├── [ 25M]  containerd.io_0.0.0~20230330082121~812111e-1~debian.12~bookworm_amd64.deb
            ├── [ 70K]  provenance.json
            └── [9.0M]  sbom.json
```

```
./pkg/containerd/bin
└── [4.0K]  ubuntu
    └── [4.0K]  bionic
        └── [4.0K]  amd64
            ├── [2.8M]  containerd.io-dbgsym_0.0.0~20230330082121~812111e-1~ubuntu.18.04~bionic_amd64.ddeb
            ├── [5.7K]  containerd.io_0.0.0~20230330082121~812111e-1~ubuntu.18.04~bionic_amd64.buildinfo
            ├── [1.8K]  containerd.io_0.0.0~20230330082121~812111e-1~ubuntu.18.04~bionic_amd64.changes
            ├── [ 25M]  containerd.io_0.0.0~20230330082121~812111e-1~ubuntu.18.04~bionic_amd64.deb
            ├── [ 70K]  provenance.json
            └── [5.0M]  sbom.json
```

```
./pkg/docker-engine/bin
└── [4.0K]  debian
    └── [4.0K]  bookworm
        └── [4.0K]  amd64
            ├── [8.6M]  docker-ce-rootless-extras_0.0.0~20230330133036~efc7a2a-3~debian.12~bookworm_amd64.deb
            ├── [6.4K]  docker-ce_0.0.0~20230330133036~efc7a2a-3~debian.12~bookworm_amd64.buildinfo
            ├── [1.8K]  docker-ce_0.0.0~20230330133036~efc7a2a-3~debian.12~bookworm_amd64.changes
            ├── [ 22M]  docker-ce_0.0.0~20230330133036~efc7a2a-3~debian.12~bookworm_amd64.deb
            ├── [ 61K]  provenance.json
            └── [ 11M]  sbom.json
```

```
./pkg/docker-engine/bin
└── [4.0K]  ubuntu
    └── [4.0K]  jammy
        └── [4.0K]  amd64
            ├── [8.6M]  docker-ce-rootless-extras_0.0.0~20230330133036~efc7a2a-3~ubuntu.22.04~jammy_amd64.deb
            ├── [7.8K]  docker-ce_0.0.0~20230330133036~efc7a2a-3~ubuntu.22.04~jammy_amd64.buildinfo
            ├── [1.8K]  docker-ce_0.0.0~20230330133036~efc7a2a-3~ubuntu.22.04~jammy_amd64.changes
            ├── [ 22M]  docker-ce_0.0.0~20230330133036~efc7a2a-3~ubuntu.22.04~jammy_amd64.deb
            ├── [ 61K]  provenance.json
            └── [7.0M]  sbom.json
```

File: pkg/compose/verify.Dockerfile [hunk 1]
```diff
@@ -53,6 +54,7 @@ FROM ${PKG_BASE_IMAGE} AS verify-rpm
 COPY --from=xx / /
 ARG PKG_RELEASE
 ARG PKG_DISTRO
+ARG PKG_ID
```
Comment (thaJeztah) [inline pkg/compose/verify.Dockerfile:57]: Wondering if `PKG_DISTRO_ID` (to indicate these are related to the distro) would be clearer (I guess same for `PKG_DISTRO_SUITE`)

It's a bit longer of course 🤔

Comment (crazy-max) [reply pkg/compose/verify.Dockerfile]: Sure, should be good.

**Review by thaJeztah — APPROVED** [review]
SGTM

**Review by ankit-gautam23 — APPROVED** [review]
LGTM

**Pull request merged**